sources/packages/forms/src/form-definitions/uploadstudentdocumentsaest.json

```diff
@@ -223,7 +223,7 @@
               "value": ""
             }
           ],
-          "content": "Rename the file with <span class=\"font-weight-bold\">“AEST_[DocumentName]”</span>. This will help students identify in their account which documents were uploaded from the ministry.\n",
+          "content": "Rename the file with <span class=\"font-weight-bold\">“[DocumentName]”</span>. This will help students identify in their account which documents were uploaded from the ministry.\n",
```

Comment:
Please change the message as provided by @hellolynn-tbtb 

![image](https://user-images.githubusercontent.com/61259237/222858016-9c2fb43b-ed80-4863-99c4-07070dd412aa.png)

Reply:
Completed.